src/rust/src/expr/string.rs

```diff
@@ -0,0 +1,390 @@
+use crate::conversion::Wrap;
+use crate::{PlRExpr, RPolarsErr};
```

Comment:
We can use `prelude::*` here.

```suggestion
use crate::{prelude::*, PlRExpr, RPolarsErr};
```

https://github.com/eitsupi/neo-r-polars/blob/800cdb866057b7f45292119b2b03ba99eed474c8/src/rust/src/prelude.rs#L1-L3